### PR TITLE
BUG Restore travis artifacts

### LINKED ISSUE #6046: BUG Travis artifact upload broken due to removal of travis-support
We will need to develop another script (inline within .travis.yml, or maybe a separate bashfile) that can be substituted for doing travis artifcact upload on behat error.

https://github.com/silverstripe/silverstripe-framework/blob/08a39f5f93cb546abb6ffdccb7fe4605a96a8ece/.travis.yml#L65

### PR DESCRIPTION
**Pull request by tractorcow**
Fixes #6046

### CODE REVIEW

File: tests/behat/travis-upload-artifacts.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+$opts = getopt('', array(
+	'artifacts-path:',
+	'target-path:',
+	'if-env:',
+	'artifacts-base-url:',
+));
+
+// --if-env=BEHAT_TEST means that this script will only be executed if the given environment var is set
+if (empty($opts['if-env'])) {
+	echo "--if-env option is mandatory";
```
Comment (tractorcow) [inline tests/behat/travis-upload-artifacts.php]: Enforced --if-env to prevent this script being invoked outside of the correct context.

**Comment by sminnee** [comment]
Good to merge when green

**Comment by tractorcow** [comment]
Behat tests currently fail on CMS occasionally. Not due to this PR, which is the point of getting artifacts fixed again. :) I suggest we fix this even with the failure.

**Comment by sminnee** [comment]
Merging as this will help diagnose the intermittent failure still present in the CMS build.

**Pull request merged**